frame.js

```diff
@@ -0,0 +1,44 @@
+class Frame {
+  constructor() {
+    this.roll1 = 0;
+    this.roll2 = 0;
+    this.roll3 = 0;
+    this.total = 0;
+  }
+
+  getRoll1() {
```

Comment:
Nice separation of responsibilities between getting and setting the 3 different types of rolls!